ghost/email-service/test/batch-sending-service.test.js

```diff
@@ -664,7 +726,7 @@ describe('Batch Sending Service', function () {
             });
 
             assert.equal(result, false);
-            sinon.assert.calledTwice(errorLog);
+            sinon.assert.notCalled(errorLog);
```

Comment:
So in this case it is not logged because the mailgun provider would have logged the error

Reply:
In the past it was logged twice